src/main/java/org/maxgamer/quickshop/Shop/DisplayItem.java

```diff
@@ -14,17 +14,17 @@
 import java.util.ArrayList;
 import java.util.List;
 
-/**
-* constant literal to check if item is quickshop display item
-*/
-static final String QUICKSHOP_DISPLAY_ITEM = "§q§s§ §d§i§s§p§l§a§y";
-
 /**
  * @author Netherfoam A display item, that spawns a block above the chest and
  * cannot be interacted with.
  */
 public interface DisplayItem {
 
+    /**
+    * constant literal to check if item is quickshop display item
+    */
+    static final String QUICKSHOP_DISPLAY_ITEM = "§q§s§ §d§i§s§p§l§a§y";
```

Comment:
Some special build server have bad support for color code. For compatibility i recommend use normal text.

(it not appears on 1.14 but before on 1.13 some legacy a magic build.)